tox.ini

```diff
@@ -2,7 +2,7 @@
 envlist = {py38}
 
 [testenv]
-passenv = GOOGLE_APPLICATION_CREDENTIALS TEST_PROJECT_NAME
+passenv = TEST_PROJECT_NAME GOOGLE_APPLICATION_CREDENTIALS
```

Comment:
Why change the order? You no longer have GOOGLE_APPLICATION_CREDENTIALS in the test VM so can't this be removed?

Reply:
This is accidental, I'll revert it

Reply:
I think it does actually need it for the tests though - the auth step in the workflow sets that variable but it needs to be passed into tox